CHANGELOG.md

```diff
@@ -4,6 +4,11 @@
 - Support for vars in top level ansible watches. ([#2147](https://github.com/operator-framework/operator-sdk/pull/2147))
 - Support for `"ansible.operator-sdk/verbosity"` annotation on Custom Resources watched by Ansible based operators to override verbosity on an individual resource. ([#2102](https://github.com/operator-framework/operator-sdk/pull/2102))
 - Support for relative helm chart paths in the Helm operator's watches.yaml file. ([#2287](https://github.com/operator-framework/operator-sdk/pull/2287))
+- Added new `--bundle` flag to the `operator-sdk scorecard` command to support bundle validation testing. ([#1916](https://github.com/operator-framework/operator-sdk/pull/1916)
+- Added new `--bundle` flag to the `operator-sdk scorecard` command to support bundle validation testing using the validation API (https://github.com/operator-framework/api). ([#1916](https://github.com/operator-framework/operator-sdk/pull/1916)
```

Comment:
Could not both be the same line? Why are both required?

Reply:
good catch, I removed the duplicate line.